### ISSUE TITLE
Review Service Design's form guide for VFS teams

### ISSUE THREAD

**Issue by strelichl** [department-of-veterans-affairs/va.gov-team#77654]
## Issue Description

Dan Turner from the Service Design Team met with Laura, Megan and I to share parts of the guide they're building for VFS teams working on digitizing forms (very exciting).
> “Even small thoughts sooner (deep process thoughts could be another discussion) would be great. We just don’t want this to become a ticket that carries over into the next sprint, so getting your feedback and having enough time to make the changes to the Phase 3 page before next Wednesday is our constraint.”

### Notes on feedback
- They’d like to get a review on 4 of the sections they have so far, from each CAIA discipline. They're working in Confluence so we can only see the PDF versions.
- If you use the comment feature and save as a new pdf (with your initials at the end) that works, we'll just have to hand back 1 version for each discipline for them to combine.
- Phase 3 and 5 have notes at the top for what feedback they're looking for, and there's a few places (like in Phase IV) where they've noted "WHAT DOES CAIA THINK." In general, they want teams to know when to involve and how to work with CAIA (definitely overlaps with our collab playbook conversations).

### Confluence Links
[Stages for forms digitization ](https://vfs.atlassian.net/wiki/spaces/SDT/pages/2900951393/Stages+for+forms+digitization)
- [Phase 1](https://vfs.atlassian.net/wiki/spaces/SDT/pages/2924609560/Phase+I+Before+starting+digitization+work)
- [Phase 2](https://vfs.atlassian.net/wiki/spaces/SDT/pages/2933391361/Phase+II+Discovery+and+planning)
- [Phase III](https://vfs.atlassian.net/wiki/spaces/SDT/pages/2965766145/Phase+III+Initial+digitization)
- [Phase IV](https://vfs.atlassian.net/wiki/spaces/SDT/pages/2970943489/Phase+IV+Design+feedback+and+testing)
- [Phase V](https://vfs.atlassian.net/wiki/spaces/SDT/pages/2969272321/Phase+V+External+Review+and+Release+Prep)
- [Phase VI](https://vfs.atlassian.net/wiki/spaces/SDT/pages/2969272321/Phase+V+External+Review+and+Release+Prep)
    - [Phase VI.I ](https://vfs.atlassian.net/wiki/spaces/SDT/pages/2900590967/I.+Before+starting+form+digitization+work)
    - [Phase VI.II](https://vfs.atlassian.net/wiki/spaces/SDT/pages/2900492564/II.+Initial+research+and+planning)
    - [Phase VI.III](https://vfs.atlassian.net/wiki/spaces/SDT/pages/2900787514/III.+Initial+digitization)
    - [Phase VI.IV](https://vfs.atlassian.net/wiki/spaces/SDT/pages/2900787526/IV.+Getting+feedback+and+testing)
    - [Phase VI.V](https://vfs.atlassian.net/wiki/spaces/SDT/pages/2900459822/V.+Release+prep)
    - [Phase VI.VI](https://vfs.atlassian.net/wiki/spaces/SDT/pages/2900885783/VI.+Launch)
    - [Phase VI.VII](https://vfs.atlassian.net/wiki/spaces/SDT/pages/2900328906/VI.+Post-launch+and+monitoring)

---

```[tasklist]
### CAIA Tasks
- [ ] Content review (Laura)
- [x] IA review (Kristin) ticket [#17394](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17394)
- [x] A11y review (Sara/Jamie)
- [x] Resolved PDF issue with [Word download option](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77654#issuecomment-1981542444)
- [ ] Marked up PDFs (or Word Docs - which are now available) shared back to Dan (Lily)
```
```[tasklist]
## Acceptance Criteria
- [ ] CAIA feedback shared with Service Design
```

**Comment by coforma-jamie** [comment]
I have access to Confluence and gave comments directly there. They've already made changes based on my comments. I can save the latest versions as PDFs and share with the team in Slack, just let me know @strelichl :)

**Comment by sara-amanda** [comment]
# Confluence

CAIA a11ys have access to Confluence, and it appears we have an option to export into a Word (.doc) file, and that it will render correctly in Word for the Content Team to edit in Word, due to no access in Confluence.  @strelichl @coforma-terry 

## Steps to Download in Confluence
### Word Version
The following steps are also represented in the image, below. 
1. Click on the three dots, in the top right-hand corner. 
2. Choose "Export" from the drop-down menu that appears. 
3. Select "Export to Word"
4. The file that downloads needs to be opened in Word to render correctly. 

![More-Actions-Menu-in-Confluence](https://github.com/department-of-veterans-affairs/va.gov-team/assets/124186314/82722650-0413-4a85-b380-9bc5d836cb40)
